test/HarpoonKrakenTests.cpp

```diff
@@ -134,7 +193,7 @@ TEST_F(HarpoonKrakenTests, SendThirtyDataChunksReceivedMethods) {
    Harpoon::Spear status = client.Aim(location);
    EXPECT_EQ(status, Harpoon::Spear::IMPALED); 
 
-   for (int i = 0; i < 30; ++i){
+   for (int i = 0; i < 30; i++){
```

Comment:
It doesn't matter but a change from pre increment to post increment doesn't make any sense.